bedrock/settings/base.py

```diff
@@ -952,7 +955,7 @@ def lazy_email_backend():
 }
 
 MOFO_SECURITY_ADVISORIES_PATH = config('MOFO_SECURITY_ADVISORIES_PATH',
-                                       default=path('..', 'mofo_security_advisories'))
+                                       default=path('mofo_security_advisories'))
```

Comment:
I'm not opposed to this change or to the removal in `.bedrock_demo_env`, but why is it part of this commit and PR?